packages/askar/src/wallet/AskarBaseWallet.ts

```diff
@@ -28,6 +28,7 @@ import {
   KeyBackend,
   KeyType,
   utils,
+  expandIfPossible,
```

Comment:
If we export i think we should have a more unique name. expandPublicKeyIfPossible maybe?